README.md

```diff
@@ -805,14 +806,14 @@ DDD
 FFF
 ```
 
-Note that the order of the numbers must be in ascending order.
-Now, on its own, this looks like a feature that is just a slight development of the `-l` option.
+Note that the order of the numbers must be ascending.
+Now, on its own, this looks like a feature that is just a slight improvement of the `-l` option.
 
-However, the breakthrough of this feature is that **the pipeline obtains identical standard input as `teip`**.
-Thus, it can output any number using not only `seq` and `echo`, but also commands such as `grep`, `sed`, and `awk`, which process the standard input.
+However, the breakthrough feature of `-e` is that **the pipeline obtains identical standard input as the main `teip` command**.
+Thus, it generate output using not only `seq` and `echo`, but also commands such as `grep`, `sed`, and `awk`, which process the standard input.
```

Comment:
generate -> generates

Reply:
Fixed.  "can generate" might be even better.